app/uat_actions/uat_actions/plate_information.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
```

Comment:
Stuff like this makes me wonder what's the difference between the UAT actions and the API? Should the integration tests be using the API, and we should be enhancing that?